packages/pwa/app/components/product-scroller/index.jsx

```diff
@@ -98,6 +98,15 @@ const ProductScroller = forwardRef(
                                             {...(typeof productTileProps === 'function'
                                                 ? {...productTileProps(product)}
                                                 : {...productTileProps})}
+                                            imageProps={{
+                                                sizes: {
+                                                    base: '70vw',
+                                                    md: '40vw',
+                                                    lg: '30vw',
+                                                    '2xl': '489px'
+                                                },
```

Comment:
Dumb question: why do we use px for `2xl` while the rest is `vw`?

Reply:
@alexvuong that's because for `2xl`, the images no longer growing bigger and they're fixed size. So I was able to use a px unit for them.

There's a pattern to our page layout design in general: at '2xl', the main content no longer stretches because there's a max-width assigned to the page. As a result, the images inside also do not grow anymore.